src/Apps/Order/orderRoutes.tsx

```diff
@@ -89,7 +89,7 @@ export const orderRoutes: AppRouteConfig[] = [
   {
     // TODO: Still need order2?
     path: "/order(2|s)/:orderID",
-    layout: "ContainerOnly",
+    layout: "None",
```

Comment:
Might be worthwhile adding a comment above this first instance noting that we provide the layout elsewhere, as the app can appear within a modal or as a page. 

(Meaning, on first glance it seems confusing that we have an app without a layout, when it _seems_ to have a layout upon viewing the page.)

Reply:
Yes, it is a bit confusing. Added a comment 👍